src/app.js

```diff
@@ -1 +1,95 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+function createULElementWithId(){
+    const unorderedList = document.createElement('ul');
+    unorderedList.setAttribute('id', 'countryList');
+    document.body.appendChild(unorderedList);
+    return unorderedList;
+}
+
+function setContinentColor(continentName){
+    switch(continentName){
+        case 'Africa':
+            return 'blue';
+            break;
```

Comment:
Na een return hoef je geen break te doen:
```suggestion
            return 'blue';
```